src/dos/program_intro.cpp

```diff
@@ -24,14 +24,30 @@
 
 void INTRO::WriteOutProgramIntroSpecial()
 {
-    WriteOut(MSG_Get("PROGRAM_INTRO_SPECIAL"), MMOD2_NAME,
-                MMOD2_NAME, PRIMARY_MOD_NAME, PRIMARY_MOD_PAD,
-                PRIMARY_MOD_NAME, PRIMARY_MOD_PAD, PRIMARY_MOD_NAME,
-                PRIMARY_MOD_PAD, PRIMARY_MOD_NAME, PRIMARY_MOD_PAD,
-                PRIMARY_MOD_NAME, PRIMARY_MOD_PAD, PRIMARY_MOD_NAME,
-                PRIMARY_MOD_PAD, PRIMARY_MOD_NAME, PRIMARY_MOD_PAD,
-                PRIMARY_MOD_NAME, PRIMARY_MOD_PAD, PRIMARY_MOD_NAME,
-                PRIMARY_MOD_PAD, MMOD2_NAME);
+	WriteOut(MSG_Get("PROGRAM_INTRO_SPECIAL"),
+	         MMOD2_NAME, // Alt, for fuilscreen toggle
+	         MMOD2_NAME, // Alt, for pause/unpause
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for keymapper
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for swap disk image
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for screenshot
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for sound recording
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for video recording
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for mute/unmute
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for shutdown
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for mouse capture
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for slow down
+	         PRIMARY_MOD_PAD,
+	         PRIMARY_MOD_NAME, // Ctrl/Cmd, for speed up
+	         PRIMARY_MOD_PAD,
+	         MMOD2_NAME); // Alt, for turbo
```

Comment:
Ugh... I get it why it's needed, but it's super nasty 😢

Reply:
Yes, that code is smelling! 🧀 
I think a better approach would be to split the huge message into one-per-line. (Although, I'm reluctant to do that in this PR; perhaps when we move to gettext).

Reply:
Sure, that's a separate thing, I just could help making a comment about it 😅